### PR TITLE
Initial workflow for attackerkb vulnerabilities from microsoft teams

### PR DESCRIPTION
**Pull request by zyoutz-r7**
## Requirements

Workflow contributors, verify you have completed the following items by checking them off:

### Checklist
- [ ] Workflow verified as working in a test
- [ ] Workflow bundle includes `Name_of_Workflow.icon` file, `help.md` file, `workflow.spec.yaml` file, and `screenshots` directory
- [ ] Workflow title should follow general [Title of Articles](http://grammar.yourdictionary.com/capitalization/rules-for-capitalization-in-titles.html) format
- [ ] Workflow includes a detailed Description
- [ ] Workflow step names follow [Title of Articles](http://grammar.yourdictionary.com/capitalization/rules-for-capitalization-in-titles.html) format
- [ ] Triggers, Decisions, and other workflow steps include descriptions where needed
- [ ] Workflow includes at least one `Final Report` artifact that summarizes workflow execution details
- [ ] Workflow bundle (directory) is named after the title of the workflow in snake case with first letters capitalized, e.g. `workflows/Name_of_Workflow/`
- [ ] `help.md` file includes Description, Key Features, Requirements, Setup, Technical Details, Version History, and References
- [ ] `workflow.spec.yaml` file includes `products`, `name`, `title`, `description`, `use_cases` tags, `keywords` tags, `source_url`, and the `name` for each screenshot matches a PNG image in the `screenshots` directory

### Assessment

Post your results with screenshots in this PR using the following style:

```
### Screenshots

Screenshots help your reviewers provide valuable feedback on your contribution. To add a screenshot here, simply drag and drop the PNG files from a file explorer.

#### Workflow Builder

<Insert Workflow Image Here>

#### Job

<Insert Job Image Here>

#### Artifact

<Insert Artifact Image Here>
```

### CODE REVIEW

File: workflows/Lookup_Exploit_with_AttackerKB_from_Microsoft_Teams/help.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+Sample Trigger Commands:
+
+`!vuln-check CVE-2020-0674`
+
+# Key Features
+
+* Lookup vulnerability exploit information from Microsoft Teams
+* Find all vulnerabilities that match a search term
+* Get description, disclosure date, attack vector, and vulnerable software versions in an easy to consume Slack message
+
```
Comment (sengleson-r7) [inline workflows/Lookup_Exploit_with_AttackerKB_from_Microsoft_Teams/help.md:14]:
```suggestion
* Get description, disclosure date, attack vector, and vulnerable software versions in an easy to consume Microsoft Teams message

```

File: workflows/Lookup_Exploit_with_AttackerKB_from_Microsoft_Teams/workflow.spec.yaml [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  license_url: https://github.com/rapid7/insightconnect-workflows/blob/master/LICENSE
+  screenshots:
+  - name: execution1.png
+    title: Workflow Execution in Microsoft Teams, Single Result
+  - name: execution2.png
+    title: Workflow Execution in Microsoft Teams, Multiple Results
+  - name: workflow.png
+    title: Workflow View
+  - name: job.png
+    title: Completed Job In Microsoft Teams
```
Comment (sengleson-r7) [inline workflows/Lookup_Exploit_with_AttackerKB_from_Microsoft_Teams/workflow.spec.yaml:25]: I think we're missing screenshot files here?

File: workflows/Lookup_Exploit_with_AttackerKB_from_Microsoft_Teams/help.md [hunk 3]
```diff
@@ -0,0 +1,68 @@
+# Description
+
+Lookup an AttackerKB exploit to receive exploit details, including a description, exploit references, vulnerable software versions, attacker value, and exploitability score from Microsoft Teams.
+
+Sample Trigger Commands:
+
+`!vuln-check CVE-2020-0674`
```
Comment (sengleson-r7) [inline workflows/Lookup_Exploit_with_AttackerKB_from_Microsoft_Teams/help.md]: should be !lookup-vuln

File: workflows/Lookup_Exploit_with_AttackerKB_from_Microsoft_Teams/help.md [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+After updating the steps with your Microsoft Teams configuration, activate the workflow in order to trigger it.
+
+## Usage
+
+*This workflow will only trigger in the channel specified in the Microsoft Teams workflow steps.*
+
+To run the workflow, in the channel you are monitoring send a message to Insight Connect using `!lookup-vuln <search term>`.
+
+For example:
+* `!vuln-check CVE-2020-0674`
```
Comment (sengleson-r7) [inline workflows/Lookup_Exploit_with_AttackerKB_from_Microsoft_Teams/help.md]: should be !lookup-vuln

File: workflows/Lookup_Exploit_with_AttackerKB_from_Microsoft_Teams/Lookup_Exploit_with_AttackerKB_from_Microsoft_Teams.icon [hunk 5]
```diff
@@ -0,0 +1,2272 @@
...
+            },
+            "defaultInputJSONSchema": null,
+            "defaultOutputJSONSchema": null,
+            "outputJSONSchema": null,
+            "defaultImageData": "https://us.cdn-assets.connect.insight.rapid7.com/step-type-icons/automated-decision.svg",
+            "connectionType": "NONE"
+          },
+          "a33e0278-63ef-423e-a24c-14693f93e4ef": {
+            "nodeId": "a33e0278-63ef-423e-a24c-14693f93e4ef",
+            "name": "Exploit Not Found Notificiation",
```
Comment (hwilsey-r7) [inline workflows/Lookup_Exploit_with_AttackerKB_from_Microsoft_Teams/Lookup_Exploit_with_AttackerKB_from_Microsoft_Teams.icon:1276]:
```suggestion
            "name": "Exploit Not Found Notification",
```

File: workflows/Lookup_Exploit_with_AttackerKB_from_Microsoft_Teams/Lookup_Exploit_with_AttackerKB_from_Microsoft_Teams.icon [hunk 6]
```diff
@@ -0,0 +1,2272 @@
...
+              "slugName": "microsoft_teams",
+              "slugVersion": "2.0.3",
+              "imageData": "https://us.cdn-assets.connect.insight.rapid7.com/icons/rapid7/microsoft_teams/2.0.3/icon.png"
+            },
+            "identifier": "new_message_received",
+            "continueOnFailure": false,
+            "isDisabled": false,
+            "parameters": {
+              "input": {
+                "channel_name": "changeme",
```
Comment (zyoutz-r7) [inline workflows/Lookup_Exploit_with_AttackerKB_from_Microsoft_Teams/Lookup_Exploit_with_AttackerKB_from_Microsoft_Teams.icon:1076]:
```suggestion
                "channel_name": "change_me",
```

**Pull request merged**